- need_name: Greet
  intention: EnterSchool
  IDtype: Teacher
  person_emotion: Happy
  rob_emotion: [0.5, 0, 0, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0
  weight: 0.9
- need_name: Greet
  intention: EnterSchool
  IDtype: Student
  person_emotion: Happy
  rob_emotion: [0.5, 0, 0, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0
  weight: 0.88
- need_name: Greet
  intention: EnterSchool
  IDtype: Parent
  person_emotion: Happy
  rob_emotion: [0.5, 0, 0, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0
  weight: 0.86 
  
- need_name: StopStranger
  intention: EnterSchool
  IDtype: Stranger
  person_emotion: Happy
  rob_emotion: [0, 0, 0, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0.0
  weight: 1 


- need_name: MeasureTempareture
  intention: EnterSchool
  IDtype: Teacher
  person_emotion: Happy
  rob_emotion: [0.5, 0, 0, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0
  weight: 0.75
- need_name: MeasureTempareture
  intention: EnterSchool
  IDtype: Student
  person_emotion: Happy
  rob_emotion: [0.5, 0, 0, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0
  weight: 0.75
- need_name: MeasureTempareture
  intention: EnterSchool
  IDtype: Parent
  person_emotion: Happy
  rob_emotion: [0.5, 0, 0, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0
  weight: 0.75



- need_name: ParentIdentity
  intention: EnterSchool
  IDtype: Parent
  person_emotion: Happy
  rob_emotion: [0.5, 0, 0, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0
  weight: 0.80 



- need_name: Anwser
  intention: Query
  IDtype: Teacher
  person_emotion: Happy
  rob_emotion: [0.5, 0, 0, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0.05
  weight: 0.85 
- need_name: Anwser
  intention: Query
  IDtype: Student
  person_emotion: Happy
  rob_emotion: [0.5, 0, 0, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0.05
  weight: 0.85  
- need_name: Anwser
  intention: Query
  IDtype: Parent
  person_emotion: Happy
  rob_emotion: [0.5, 0, 0, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0.05
  weight: 0.85  



- need_name: KeepOrder
  intention: Uncooperate
  IDtype: Teacher
  person_emotion: Happy
  rob_emotion: [0, 0, 0.5, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0.0
  weight: 0.85  
- need_name: KeepOrder
  intention: Uncooperate
  IDtype: Student
  person_emotion: Happy
  rob_emotion: [0, 0, 0.5, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0.0
  weight: 0.85  
- need_name: KeepOrder
  intention: Uncooperate
  IDtype: Parent
  person_emotion: Happy
  rob_emotion: [0, 0, 0.5, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0.0
  weight: 0.85  




- need_name:  Remind
  intention: SendWord
  IDtype: Teacher
  person_emotion: Happy
  rob_emotion: [0, 0, 0.5, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0.0
  weight: 0.85  
- need_name:  Remind
  intention: SendWord
  IDtype: Student
  person_emotion: Happy
  rob_emotion: [0, 0, 0.5, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0.0
  weight: 0.85  
- need_name:  Remind
  intention: SendWord
  IDtype: Parent
  person_emotion: Happy
  rob_emotion: [0, 0, 0.5, 0, 0, 0, 0, 0 ]
  rob_status: [1, 1, 1, 1, 1, 1, 1, 0 ]
  time: 0.0
  weight: 0.85  

